src/main/kotlin/eu/europa/ec/eudi/openid4vci/internal/RequestIssuanceImpl.kt

```diff
@@ -118,34 +120,41 @@ internal class RequestIssuanceImpl(
                 ) {
                     "The credential identifier passed is not valid or unknown"
                 }
-                identifierBasedRequest(credentialConfigurationId, credentialId, proofFactory)
+                identifierBasedRequest(credentialConfigurationId, credentialId, proofFactory, includeEncryptionSpec)
             }
         }
 
     private fun formatBasedRequest(
         credentialConfigurationId: CredentialConfigurationIdentifier,
         claimSet: ClaimSet?,
         proofFactory: ProofFactory?,
+        includeEncryptionSpec: Boolean,
```

Comment:
Minor: Perhaps `includeEncryptionSpec` can be renamed to the more generic `partOfBatch`
In this case the 
```kotlin
val includeEncryptionSpec = !partOfBatch
```
IMO this express better the semantics, but requires to change the caller side